.travis.yml

```diff
@@ -1,30 +1,32 @@
 language: php
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
+  - 5.4
+  - 5.5
+  - 5.6
+  - hhvm
 
 env:
-  - SYMFONY_VERSION=2.5.*
-  
+  - SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+
 matrix:
-  allow_failures:
-    - env: SYMFONY_VERSION=dev-master
   include:
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.5
       env: SYMFONY_VERSION=2.3.*
     - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
+      env: SYMFONY_VERSION=2.6.*
     - php: 5.5
-      env: SYMFONY_VERSION=dev-master
-    
-
+      env: SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+    - php: 5.5
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
missing allowed failures for:

* Symfony 2.8 (which should not set the deprecations helper to weak btw, so we can catch new deprecations in 2.8)
* Symfony 3.0
* PHP nightly